src/core/src/signature.rs

```diff
@@ -448,10 +448,8 @@ impl Signature {
     pub fn name(&self) -> String {
```

Comment:
since `r0.18.0` is not released yet, do we want to make this
`pub fn name(&self) -> Option<String> {`
instead? This wil change calls around in other places, but seems to communicate better that the name exists or not instead of getting a `.name()` call that might return `""`

Reply:
ooh, I like it! let me give it a try 🙇

Reply:
https://github.com/sourmash-bio/sourmash/pull/3434/commits/34dba548d24c1ebdebf62a96cdfc2b85a9c2527f